samples/highcharts/accessibility/internationalization/demo.html

```diff
@@ -0,0 +1,34 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/stock/modules/data.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<link rel="stylesheet" type="text/css" href="//github.com/downloads/lafeber/world-flags-sprite/flags32.css" />
+
+<fieldset>
+    <legend>Choose language:</legend>
+        <div class="flags" aria-checked="true">
+            <input type="radio" id="en" name="languages" value="en" checked>
+            <label for="en">
+                <span class="f32"><span id="flag" class="flag us" aria-label="The American flag"></span>
```

Comment:
I changed my mind... The `aria-label`s should probably just say "English", "Norwegian", "German", since the user does not get any indication that these are images, and such the "flag" part might just confuse them